telegram/_replykeyboardmarkup.py

```diff
@@ -114,8 +114,9 @@ def __init__(
         self.one_time_keyboard = bool(one_time_keyboard)
         self.selective = bool(selective)
         self.input_field_placeholder = input_field_placeholder
-
-        self._id_attrs = (self.keyboard,)
+        # ensure keyboard is notnull before assigning
+        if self.keyboard:
+            self._id_attrs = (self.keyboard,)
```

Comment:
I don't think this check is necessary. `keyboard` is a mandatory argument and if the user passes something that's not a list of lists of buttons, that's their fault.
Additionally, the sanity check in line 95 alread checks the "list of lists" part. So if we get here, `self.keyboard` is `[[]]` in the worst case, which will pass the `if self.keyboard` check.

Reply:
Right. Yeah the check at 95 was there, but I thought for ensuring the null check would be necessary. 

But thanks to your comment, I just reviewed and realized it doesn't get to null anywhere. Sure will do changes and let you know

Reply:
And regarding conflicts there is a huge number of conflicts showing up now but wasn't there when I pushed yesterday.. Is it for me alone?

Reply:
just pull from v14 into this branch, it should be fine :)

Reply:
Yes, I have tried to do it. I was prompted to create a PR and I did so, but now I am stuck here. Could you pls give me a solution so that I have the latest V14? Thanks in advance! 

P.S: Due to my access issue I was unable to login to Github and hence the delay. 

Here's the screen of the PR that I created when prompted.
![image](https://user-images.githubusercontent.com/36819505/147439270-55a12ac9-ac7a-427f-a6a6-e9ad924ff38c.png)